### DISCUSSION TITLE
Run on desktop?

### DISCUSSION THREAD

**Discussion by FitzM** [deckerst/aves#904] [General]
Hello.  I'm willing to do the work for it but I wanted to check first , is there any reason why this couldn't/wouldn't  work well on a desktop linux system?

**Comment by deckerst** [comment]
Despite being an app made with Flutter, there's a lot of logic done on the Android platform side (from Android itself, the various external libraries and custom code to tie it all). Also, the current way to discover and access media is intimately linked to the way media files are registered on Android. On desktop, file access would be way easier, but discovery would be something to implement.

So yes, with work, it's technically possible to turn Aves into a desktop app too, but it's not trivial.

> I'm willing to do the work for it

Not sure it's what you meant, but just so you know this repo does not accept PRs.

**Reply by FitzM** [reply]
Nope I said it because I didn't want anyone to think it was a feature request. I was planning on forking it and trying to make it work for desktop.
